containers/event-db-migrations/entry.sh

```diff
@@ -133,15 +132,18 @@ fi
 # Run migrations
 echo ">>> Running migrations..."
 export DATABASE_URL="postgres://${DB_USER}:${DB_USER_PASSWORD}@${DB_HOST}:${DB_PORT}/${DB_NAME}"
+echo "$DATABASE_URL"
 ./refinery migrate -e DATABASE_URL -c ./refinery.toml -p ./migrations
 
 # Add historic data from previous funds
-export PGUSER="${DB_USER}"
-export PGPASSWORD="${DB_USER_PASSWORD}"
-
 while IFS= read -r -d '' file; do
     echo "Adding historic data from $file"
     psql -f "$file"
 done < <(find ./historic_data -name '*.sql' -print0 | sort -z)
+# Add test data
```

Comment:
Have you tried testing this if `./test_data` doesn't exist? Since the `-e` opt is set at the top, if `find` returns a non-zero exit code it may terminate the script early.

Reply:
yes, it works in both sides, if test_data or historic_data does not exist